renku/cli/dataset.py

```diff
@@ -420,7 +420,13 @@ def prompt_tag_selection(tags):
     return None
 
 
-@click.group(invoke_without_command=True)
+@click.group()
+def dataset():
+    """Dataset commands."""
+    pass
+
+
+@dataset.command("list")
```

Comment:
Should we use `ls` instead of `list`? It is shorter and is in par with `renku dataset ls-files`.

Reply:
It would be good if we could have command aliases out of the box but it has some hassles and I'm not sure if it worths the effort: https://stackoverflow.com/questions/46641928/python-click-multiple-command-names

Reply:
https://github.com/SwissDataScienceCenter/renku-python/issues/1530